cmd/notify-mailer/main.go

```diff
@@ -110,7 +110,7 @@ func (m *mailer) run() error {
 	startTime := m.clk.Now()
 
 	for i, dest := range destinations {
-		m.printStatus(dest, i, len(destinations), startTime)
+		m.printStatus(dest, i, len(destinations)-1, startTime)
```

Comment:
Just to be incredibly pedantic, `i+1` feels like it'd make a bit more sense than `len(destinations)-1`.

Reply:
(i.e. `Message 1 of 1` rather than `Message 0 of 0`)

Reply:
WFM. I agree that reads better if you are a `H U M A N   B E I N G` :robot:

Reply:
Fixed in 7f626a1

Reply:
![](https://i.redd.it/v7pz136wgf611.jpg)